modules/olm-enabling-operator-restricted-network.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+[id="olm-enabling-operator-for-restricted-network_{context}"]
+= Enabling your Operator for restricted network environments
+
+As an Operator author, you must meet additional requirements for your Operator
+to run properly in {product-title} clusters that are in a restricted network
+environment:
+
+* List related images required by the Operator.
+* Reference the Operator image in the related images.
+* Reference all specified images by a digest (SHA) and not by a tag.
+
+.Prerequisites
+
+* An Operator project with a CSV
+
+.Procedure
+
+. Operators often require use of other container images to perform their functions
+as Operators. In your Operator's CSV, define a list of any such _related images_:
++
+[source,yaml]
+----
+kind: ClusterServiceVersion
+metadata:
+  name: etcd-operator
+spec:
+[...]
+  relatedImages: <1>
+  - name: default
+    image: quay.io/coreos/etcd@sha256:12345 <2>
+    annotation: default
+  - name: etcd-2.1.5
+    image: quay.io/coreos/etcd@sha256:12345 <2>
+  - name: etcd-3.1.1
+    image: quay.io/coreos/etcd@sha256:12345 <2>
+----
+<1> Create a `relatedImages` section and set the list of related images.
+<2> Specify each image by a digest (SHA), not by an image tag.
++
+These images are made available as annotations on the Operator's Deployment
+objects, so that they can be used via downward API if desired. For example:
++
+[source,yaml]
+----
+kind: Deployment
+metadata:
+  name: etcd-operator
+  annotations:
+    default: quay.io/coreos/etcd@sha256:12345
+    olm.relatedImage.etcd-2.1.5: quay.io/coreos/etcd@sha256:12345
+    olm.relatedImage.etcd-3.1.1: quay.io/coreos/etcd@sha256:12345
+----
++
+This can be particularly useful for Operators that are tightly coupled to
+another particular image.
+
+. The related images listed in the CSV must likewise contain a reference to the Operator image.
```

Comment:
@ecordell I'm trying to find more information about this step, if I even have described the requirement correctly.